source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRIBaseCardElementRenderer.h

```diff
@@ -21,12 +21,12 @@
     baseCardElement:(ACOBaseCardElement *)acoElem
          hostConfig:(ACOHostConfig *)acoConfig;
 @optional
-/// override this method for custom styling 
+/// override this method for custom styling
 /// not all renderers supports it
 - (void)configure:(UIView *)view
-         rootView:(ACRView *)rootView
-  baseCardElement:(ACOBaseCardElement *)acoElem
-       hostConfig:(ACOHostConfig *)acoConfig;
+           rootView:(ACRView *)rootView
```

Comment:
nit: this one looks weird

Reply:
didn't get it. what's the problem

Reply:
i don't know if you can fix it. if you can't, just leave it for now. you're using the clang-format config in the ios folder, right?

the clang-format usually lines up the ':'.
on that line, I couldn't see why it won't line them together.

Reply:
yes. I'm using clang-format config in the ios folder and didn't do any change on it.